cmd/lncli/cmd_pay.go

```diff
@@ -287,11 +293,13 @@ func sendPayment(ctx *cli.Context) error {
 		Dest:              destNode,
 		Amt:               amount,
 		DestCustomRecords: make(map[uint64][]byte),
+		Amp:               ctx.Bool(ampFlag.Name),
 	}
 
 	var rHash []byte
 
-	if ctx.Bool("keysend") {
+	switch {
```

Comment:
Does this need a check to ascertain that keysend and amp aren't both set?

Reply:
Yep

Reply:
Done